content/doc/developer/publishing/releasing-experimental-updates.adoc

```diff
@@ -2,9 +2,17 @@
 title: Publishing Experimental Plugin Releases
 layout: developer
 ---
-
-To simplify delivery of beta versions of plugins to interested users, the Jenkins project publishes an _experimental update center_.
-It includes alpha and beta versions of plugins, which aren't usually included in the regular update center.
+The link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin Manager tool] with link:/doc/developer/plugin-development/incrementals/[incremental] builds is often a preferred technique as it benefits the user by:
+- providing additional information about the plugins being downloaded (such as, available updates or security warnings and many more), 
+- enabling easier import into an instance of jenkins by downloading plugins and their respective dependencies into a folder,
+- making it easier to test and explore unreleased plugin versions,
+- providing the user with flexibility to create incremental versions of jenkins core or an upstream plugin with dependencies while also providing the ability to switch between and keep track of incremental versions.
```

Comment:
What does that mean? End users cannot create and distribute incrementals of core and plugins to use for others. Incremental builds are created if the so-called maven profile is specified and something triggers the use of the profile.
The creation of incremental builds is unrelated to the plugin manager tool.